### PR TITLE
Steven master utility class

### PR DESCRIPTION
**Pull request by smarquez-jpg**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/RepliesUtils.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  public static List<BlogMessage> putRepliesWithPosts(List<BlogMessage> blogMessagesAll) {
+    // Separate posts from replies from |blogMessages|.
+    // Add replies to messageReplies for the respective posts.
+    List<BlogMessage> blogMessagesParents = new ArrayList<BlogMessage>();
+    List<BlogMessage> blogMessagesReplies = new ArrayList<BlogMessage>();
+    for (BlogMessage message : blogMessagesAll) {
+      if (message.getParentID() == 0) {
+        blogMessagesParents.add(message);
+      } else {
+          blogMessagesReplies.add(message);
```
Comment (Andrewmob2703-zz) [inline portfolio/src/main/java/com/google/sps/data/RepliesUtils.java]: nit: indentation. should go 2 backspaces

**Comment by peizhaoou** [comment]
How is this different from the other PR?

**Comment by smarquez-jpg** [comment]
> How is this different from the other PR?

This one is similar to the other ones in that I factored out some functions into a utility class, except this particular utility function relies on the other utility functions I made and allows us to test the other functions in the order they were meant to be executed.

**Review by Andrewmob2703-zz — APPROVED** [review]
LGTM!
DataServlet looks much better!
Please manually test and make sure it works as expected.

**Review by martiKallas — APPROVED** [review]
LGTM

**Pull request merged**